src/core/types.ts

```diff
@@ -126,3 +126,45 @@ export type FrameMetadataType = {
  * Note: exported as public Type
  */
 export type FrameMetadataResponse = Record<string, string>;
+
+/**
+ * Ethereum Attestation Service (EAS) Attester Address
+ * The Ethereum address of the attester who created the attestation.
+ */
+type EASAttesterAddress = Address;
+
+/**
+ * Ethereum Attestation Service (EAS) Schema Uid
+ * The schema identifier associated with the EAS attestation.
+ * Note: exported as public Type
+ */
+export type EASSchemaUid = `0x${string}`;
+
+/**
+ * Ethereum Attestation Service (EAS) Attestation
+ * GraphQL response for EAS Attestation
+ * Note: exported as public Type
+ */
```

Comment:
nit. we should have it in alphabetical order.

Reply:
Also, is there a place where this is shared as standard? or is more a format we decided?